Ghidra/Features/Base/src/main/java/ghidra/app/util/viewer/field/AnnotatedStringHandler.java

```diff
@@ -95,4 +95,11 @@ public boolean handleMouseClick(String[] annotationParts, Navigatable sourceNavi
 	 * @return the example of how this is used.
 	 */
 	public String getPrototypeString();
+	
+	/**
+	 * Returns an example string of how the annotation is used
+	 * @param contained The text that may be wrapped
```

Comment:
Update the variable comment to indicate that the given value may not be null.

Reply:
I wasn't entirely sure if there was some standard practice for this, let me know if there is a better way